### ISSUE TITLE
[Request]: Is it possible to insert highlights and note excerpts corresponding to specific times into the diary?

### ISSUE THREAD

**Issue by isdamir** [bandantonio/obsidian-apple-books-highlights-plugin#47]
### Feature Request Checklist

- [X] I have searched the [existing issues](https://github.com/bandantonio/obsidian-apple-books-highlights-plugin/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement%2Cquestion) to avoid duplicates

### What feature would you like to request?

This plugin is very useful and particularly fast to use. I noticed that the weread plugin supports inserting highlights and note excerpts added at specific times into the diary for the corresponding dates, which is very convenient for reviewing based on the diary.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bandantonio** [comment]
Hi @isdamir
Did I understand you correctly that you want to embed highlights into the corresponding daily notes according to the date the highlights were taken on?
For example, if you made several highlights yesterday, you want them to appear in yesterday's note and all the today's highlights to appear in today's note?

**Comment by isdamir** [comment]
> Hi @isdamir Did I understand you correctly that you want to embed highlights into the corresponding daily notes according to the date the highlights were taken on? For example, if you made several highlights yesterday, you want them to appear in yesterday's note and all the today's highlights to appear in today's note?

Yes, this way I can review the content of the books through the diary. I have currently added reverse links to the template, which allows me to somewhat find the corresponding records from the diary.
<img width="863" alt="image" src="https://github.com/user-attachments/assets/28efadfc-ec33-4792-80ad-691dfd870c6f">

**Comment by isdamir** [comment]
<img width="951" alt="image" src="https://github.com/user-attachments/assets/67b5f5ba-3e75-4dc4-b349-2d5076a440bf">
The above is the effect of the weread plugin adding content to the diary.
